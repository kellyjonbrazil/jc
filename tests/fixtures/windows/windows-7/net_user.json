{
  "domain": "",
  "user_accounts": [
    {
      "user_name": "Administrator"
    },
    {
      "user_name": "Guest"
    },
    {
      "user_name": "user1"
    }
  ],
  "account_origin": "\\\\DESKTOP-WIN7"
}
