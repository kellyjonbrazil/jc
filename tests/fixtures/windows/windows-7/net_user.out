
User accounts for \\DESKTOP-WIN7

-------------------------------------------------------------------------------
Administrator            Guest                    user1
The command completed successfully.